src/data-provider.ts

```diff
@@ -77,6 +78,44 @@ export class FileCoverageDataProvider implements vscode.TreeDataProvider<Coverag
         }
     }
 
+    public async generateCoverage(): Promise<string> {
+        if (!vscode.workspace.workspaceFolders) {
+            this.logger.warn('Empty workspace');
+            return Promise.reject("Empty workspace");
+        }
+
+        if (!this.configStore.current.coverageCommand) {
+            this.logger.warn('No coverage command set.');
+            return Promise.reject("No coverage command set.");
+        }
+
+        const rootProjectPath = vscode.workspace.workspaceFolders[0].uri.fsPath;
```

Comment:
This won't work for workspaces with multiple folders. Don't know how to do it yet, but can't be merged this way.